python/lsst/summit/utils/efdUtils.py

```diff
@@ -308,7 +321,10 @@ async def _getEfdData(
     availableTopics = await client.get_topics()
 
     if topic not in availableTopics:
-        raise ValueError(f"Topic {topic} not in EFD schema")
+        if raiseIfTopicNotInSchema:
+            raise ValueError(f"Topic {topic} not in EFD schema")
+        else:
+            return pd.DataFrame()
```

Comment:
Do you need to ensure that the empty dataframe contains any specific columns, or has an index?

Reply:
Also, I suggest adding a warning (could be at `VERBOSE` level) stating that the topic was not in the schema and empty data is being returned.

Reply:
Ha, if only data in the EFD was that predictable! But yeah, columns don't matter, the main thing is that it doesn't raise - downstream code that's setting the "don't raise if the topic isn't there" is then responsible for dealing with the lack of data.

Reply:
By `VERBOSE` I assume you mean `DEBUG`, in which case sure 👍